packages/amplify-cli/src/index.ts

```diff
@@ -18,6 +20,13 @@ EventEmitter.defaultMaxListeners = 1000;
 // entry from commandline
 export async function run(): Promise<number> {
   try {
+    // Checks for available update on each use
+    const notifier = updateNotifier({
+      pkg,
+      updateCheckInterval: 0,
```

Comment:
I think to no to annoy the users on each execution it could be a few days interval, looking that we're releasing ~weekly. 3 or 5 days?